woocommerce/woocommerce-ios
# Delete a Downloadable file for a Product

Fixes #2758 

## Description
As part of Products M5 we would allow users to delete a downloadable file from the downloadable file screen detail.
In this PR, I implemented the action sheet that shows the `Delete` button + the action.

## Testing
1. Open a product with downloadable files.
2. Open a downloadable file detail.
3. Tap the "more" button item on the navbar, that will be visible only when you are editing a downloadable file.
4. Press the `Delete` button.
5. Update the product, and make sure that the changes will be applied correctly, removing the specified downloadable file.

## GIF
<img src="https://user-images.githubusercontent.com/495617/96268343-3d868000-0fc9-11eb-8632-145bfc86d014.gif" width=300 />

Update release notes:
